app/main.py

```diff
@@ -1,3 +1,13 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: callable) -> callable:
+    cache_lib = {}
+
+    def inner(*args) -> callable:
+        nonlocal cache_lib
```

Comment:
this statement is redundant, as dict is a mutable type

Reply:
I did that because I use a variable in a different scope. I didn't know that it's not necessary to use `nonlocal` in mutable types